ravenframework/PluginManager.py

```diff
@@ -157,10 +157,11 @@ def getEntities(entityType=None):
 
 # load plugins directory and collect plugins
 pluginsPath = os.path.join(_filePath, '..', 'plugins')
-# use "catalogue" to differentiate between "path" and "directory"
-pluginsCatalogue = os.path.abspath(os.path.join(pluginsPath, 'plugin_directory.xml'))
-# if no installed plugins, report and finish; otherwise, load plugins
-if os.path.isfile(pluginsCatalogue):
-  loadPlugins(pluginsPath, pluginsCatalogue)
-else:
-  print('PluginFactory: No installed plugins detected.')
+if os.path.isdir(pluginsPath): #If false, then probably running as pip package
```

Comment:
Do we need to do something if this check is false?

Reply:
Hm, this check will always be false when pip installed. It could also be false if someone randomly deleted the directory. So I think we don't need to do anything.